
module.exports = {

  AUTH_USER_DETAILS: 'x-auth-user-details',

  STATUS_CODE_OK: 200,
  INTERNAL_SERVER_ERROR: 500,
  VALIDATION_ERROR: 417,
  NOT_FOUND_ERROR: 404,
  AUTHORIZATION_ERROR: 401,
  FORBIDDEN_ERROR: 403,

  STATUS_SUCCESS: 1,
  STATUS_ERROR: 0,
  STATUS_WARNING: 2,
  STATUS_SET_PASSWORD: 3,
  STATUS_INVITE_SENT: 4,
  
  INTERNAL_SERVER_ERROR_MESSAGE: 'Internal server error!',
  NOT_FOUND_MESSAGE: 'Resource not found!',
  AUTHORIZATION_ERROR_MESSAGE: 'User Not Authorized!',
  PERMISSION_AUTHORIZATION_ERROR: 'You do not have the required permissions to access this resource',
}
